### PR TITLE
sync_manager: Rename Update to WebsocketUpdate

### PR DESCRIPTION
**Pull request by Beadko**
# PR Description

As per the request in PR #1229, the Update was replaced with WebsocketUpdate

Fixes # (issue)

## Type of change

Please delete options that are not relevant and add an `x` in `[]` as item is complete.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How has this been tested

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration and
also consider improving test coverage whilst working on a certain feature or package.

- [ ] go test ./... -race
- [x] golangci-lint run
- [ ] Test X

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation and regenerated documentation via the documentation tool
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally and on Github Actions/AppVeyor with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: engine/sync_manager_test.go [hunk 1]
```diff
@@ -264,52 +264,52 @@ func TestWaitForInitialSync(t *testing.T) {
 func TestSyncManagerUpdate(t *testing.T) {
```
Comment (shazbert) [inline engine/sync_manager_test.go:264]: TestSyncManagerUpdate -> TestSyncManagerWebsocketUpdate

**Review by shazbert — CHANGES_REQUESTED** [review]
Just one nit. Then looks good to go.

File: engine/sync_manager_test.go (hunk 1 above)
Comment (Beadko) [reply engine/sync_manager_test.go]: done :white_check_mark:

Comment (shazbert) [reply engine/sync_manager_test.go]: 🎉

**Review by shazbert — APPROVED** [review]
Changes ACK! Thanks Bea. 🚀

File: engine/sync_manager.go [hunk 2]
```diff
@@ -332,7 +332,7 @@ func (m *syncManager) add(k currencyPairKey, s syncBase) *currencyPairSyncAgent
 
 // Update notifies the syncManager to change the last updated time for a exchange asset pair
```
Comment (thrasher-) [inline engine/sync_manager.go:333]:
```suggestion
// WebsocketUpdate notifies the syncManager to change the last updated time for a exchange asset pair
```

**Review by thrasher- — CHANGES_REQUESTED** [review]
Just one nit and then this LGTG

**Review by shazbert — APPROVED** [review]
ch ACK!

**Comment by thrasher-** [comment]
Nit fixed!

**Pull request merged**